src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -19,10 +20,30 @@
 (react/defc GroupTable
   {:render
    (fn [{:keys [this state]}]
-     (net/render-with-ajax
-      (:groups-response @state)
-      #(this :-render-groups-table)
-      {:loading-text "Loading Groups..."}))
+     [:div {}
+      (cond
+        (:modal? @state)
```

Comment:
This seems to presume there are only two modals we'll ever show here, delete and deleting. There's already another modal, create new, so this is a faulty assumption. Change this key name, thanks.